test/Cli/Func.E2E.Tests/Commands/FuncInit/PythonInitTests.cs

```diff
@@ -0,0 +1,154 @@
+            var expectedDockerfileContent = new[] { "FROM mcr.microsoft.com/azure-functions/python:4" };
+
+            // Initialize node function app
+            var funcInitResult = funcInitCommand
+                .WithWorkingDirectory(workingDir)
+                .Execute(["--worker-runtime", "python", "--docker"]);
+
+            // Validate expected output content
+            funcInitResult.Should().ExitWith(0);
+            funcInitResult.Should().HaveStdOutContaining($"Writing Dockerfile");
+            funcInitResult.Should().FileExistsWithContent(dockerFilePath, expectedDockerfileContent);
+        }
+
+        [Fact]
+        public async Task Init_Python_App_With_Docker_Only_For_Existing_Project()
+        {
+            var workingDir = WorkingDirectory;
+            var testName = nameof(Init_Python_App_With_Docker_Only_For_Existing_Project);
+            var funcInitCommand = new FuncInitCommand(FuncPath, testName, Log ?? throw new ArgumentNullException(nameof(Log)));
+            var dockerFilePath = Path.Combine(workingDir, "Dockerfile");
+            var expectedDockerfileContent = new[] { $"FROM mcr.microsoft.com/azure-functions/python:4" };
+
+            // Initialize dotnet function app with retry helper
+            await FuncInitWithRetryAsync(testName, [".", "--worker-runtime", "python"]);
+
+            var funcInitResult = funcInitCommand
+                .WithWorkingDirectory(workingDir)
+                .Execute(["--docker-only"]);
+
+            // Validate expected output content
+            funcInitResult.Should().ExitWith(0);
+            funcInitResult.Should().HaveStdOutContaining($"Writing Dockerfile");
+            funcInitResult.Should().FileExistsWithContent(dockerFilePath, expectedDockerfileContent);
+        }
+
+        [Fact]
+        public async void Init_Python_App_Twice()
+        {
+            var workingDir = WorkingDirectory;
+            var testName = nameof(Init_Python_App_Twice);
+            var funcInitCommand = new FuncInitCommand(FuncPath, testName, Log ?? throw new ArgumentNullException(nameof(Log)));
+
+            // Initialize dotnet function app with retry helper
+            await FuncInitWithRetryAsync(testName, [".", "--worker-runtime", "python"]);
+
+            // Initialize python function app
+            var funcInitResult = funcInitCommand
+                .WithWorkingDirectory(workingDir)
+                .Execute(["--worker-runtime", "python"]);
+
+            // Validate expected output content
+            funcInitResult.Should().ExitWith(0);
+            funcInitResult.Should().HaveStdOutContaining($"requirements.txt already exists. Skipped!");
+            funcInitResult.Should().HaveStdOutContaining($"host.json already exists. Skipped!");
+            funcInitResult.Should().HaveStdOutContaining($".gitignore already exists. Skipped!");
+        }
+
+        [Fact]
+        public void Init_Python_App_ModelV1_Generates_getting_started_md()
```

Comment:
```suggestion
        public void Init_PythonApp_ModelV1_GeneratesGettingStartedDoc()
```

Lets use this kind of structure for test names i.e.

`Command_ActionTaken_Result`

Also no lowercaase in any test name